src/business-logic/services/execute-action-service.ts

```diff
@@ -100,6 +100,7 @@ export class ExecuteActionService implements ActionService {
 
   private async insertPieceAsOnAir(pieceAction: PieceAction, rundownId: string): Promise<void> {
     const piece: Piece = this.createPieceFromAction(pieceAction)
+    piece.setExecutedAt(Date.now())
```

Comment:
A test ensuring that we don't remove this line at some point would be nice :)

Reply:
Test added :)